server/repositories/cmsApi.js

```diff
@@ -15,10 +15,29 @@ class CmsApi {
     this.jsonApiClient = jsonApiClient;
   }
 
+  async lookupContent(establishmentName, contentId) {
+    const {
+      jsonapi: { resourceName, individual },
+    } = await this.jsonApiClient.getRelative(
+      `/router/prison/${establishmentName}/translate-path?path=content/${contentId}`,
+    );
+
+    return {
+      type: resourceName,
+      location: individual,
+    };
+  }
+
   async get(query) {
-    const response = await this.jsonApiClient.get(query.path());
-    const items = dataFormatter.deserialize(response);
-    return items.map(item => query.transform(item));
+    const request = query.path
+      ? this.jsonApiClient.getRelative(query.path())
+      : this.jsonApiClient.getUrl(query.url());
+    const response = await request;
+    const deserializedResponse = dataFormatter.deserialize(response);
+    if (Array.isArray(deserializedResponse)) {
```

Comment:
when retrieving a specific item referenced by the router, it will point to a single item rather than a list - this will handle the difference transparently.

We could have added a different method to `getSingle` but we didn't want to add confusion to the calling code - where the calling code would have to know which method to call based on some intrinsic aspect of the query